### PR TITLE
Add Google Maven Repository extension

### PR DESCRIPTION
**Pull request by koinzhang**
## Description

Show and search for artifacts in Google's maven repository.

Query Api of Google's maven repository from [[WanAndroid website](https://www.wanandroid.com/)](https://www.wanandroid.com)

## Screencast

Metadata-1

Metadata-2

Metadata-3

Metadata-4

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/google-maven-repository/package.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "google-maven-repository",
+  "title": "Google Maven Repository",
+  "description": "Show and search for artifacts in Google's maven repository.",
```
Comment (fe9lix) [inline extensions/google-maven-repository/package.json:5]:
```suggestion
  "description": "Show and search for artifacts in Google's Maven repository.",
```

File: extensions/google-maven-repository/package.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    "Documentation"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "show-google-maven-repository",
+      "title": "Show Google Maven Repository",
+      "icon": "show-google-maven.png",
+      "subtitle": "Google Maven Repository",
+      "description": "Show All artifact in Google's maven repository.",
```
Comment (fe9lix) [inline extensions/google-maven-repository/package.json:19]:
```suggestion
      "description": "Show All artifact in Google's Maven repository.",
```

File: extensions/google-maven-repository/package.json [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+      "mode": "view"
+    },
+    {
+      "name": "search-google-artifact",
+      "title": "Search Google Artifact",
+      "icon": "search-google-maven.png",
+      "subtitle": "Google Maven Repository",
+      "description": "Search for an artifact in Google's maven repository.",
```
Comment (fe9lix) [inline extensions/google-maven-repository/package.json:27]:
```suggestion
      "description": "Search for an artifact in Google's Maven repository.",
```

**Review by fe9lix — CHANGES_REQUESTED** [review]
Hey @koinzhang, cheers! I've left some small naming changes (mavem -> Maven). For the command "Search Google Maven Repository", I would suggest that you show a loading indicator on the list since the user first sees the empty view and then it "jumps" to the list entries.

**Comment by koinzhang** [comment]
Hi @fe9lix. Thanks for your review.
Thank you very much for the suggestion, I have submitted the commit for update.

> For the command "Search Google Maven Repository", I would suggest that you show a loading indicator on the list since the user first sees the empty view and then it "jumps" to the list entries.

Update: Fixed the bug that flashes EmptyView

Please continue to review.

**Pull request merged**